vendor/bat-native-ads/src/bat/ads/internal/segments/segments_util.cc

```diff
@@ -80,6 +81,11 @@ SegmentList GetParentSegments(const SegmentList& segments) {
   SegmentList parent_segments;
 
   for (const auto& segment : segments) {
+    if (segment.empty()) {
```

Comment:
`segments` should never be empty. Can we please instead add DCHECKs and trace the callers of this method to understand why `segments` could be empty